requirements/collections_requirements.yml

```diff
@@ -1,7 +1,7 @@
 ---
 collections:
   - name: awx.awx
-    version: 11.2.0  # almost all versions should work
+    version: 12.0.0  # 12 or higher needed for inventory token fix
```

Comment:
it may be that no changes to this file are needed anymore

but we need to clear up rebasing & testing situation

Reply:
yep, ditto - we just removed the pinned versions in this file:
https://github.com/ansible/awx/pull/7392

Reply:
Rebase is building now.